images/gcloud/Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 0.1
+VERSION = 0.2
```

Comment:
yeah, but we didn't realize until just now that 0.1 was a pretty old go + gcloud. if it was in the tag it'd be self-documenting and way more obvious.